x-pack/legacy/plugins/security/public/views/account/account.js

```diff
@@ -4,17 +4,14 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import routes from 'ui/routes';
-import template from './account.html';
-import { i18n } from '@kbn/i18n';
-import { I18nContext } from 'ui/i18n';
-import { npSetup } from 'ui/new_platform';
-import { AccountManagementPage } from './components';
 import React from 'react';
 import { render, unmountComponentAtNode } from 'react-dom';
+import { i18n } from '@kbn/i18n';
+import { npStart } from 'ui/new_platform';
+import routes from 'ui/routes';
 
 routes.when('/account', {
```

Comment:
**question** Is this route waiting on NP changes to migrate?

Reply:
Technically it should be possible to migrate it now since chromeless apps are already available in NP, but there are 2 reason why I'd do it separately:
* I want to give this new API some time to stabilize a bit
* I was planning to migrate all views in one go (it's appeared in this PR just because it uses password change component that I didn't want to expose to the LP).